server/bin/index-pbench

```diff
@@ -1,52 +1,68 @@
 #!/usr/bin/env python3
 # -*- mode: python -*-
 
-# prereqs:
-# config file: specified in CONFIG "Indexing" section
-# mapping file:
+"""Pbench indexing drriver, responsible for indexing a single pbench tar ball
+into the configured Elasticsearch V1 instance.
+
+"""
 
 from __future__ import print_function
 
-import sys, os, time, re, stat, copy
+import sys, os, re, stat, copy
 import hashlib, json, glob, csv, tarfile, shutil
 
-from urllib3 import Timeout
+from operator import itemgetter
 from datetime import datetime, timedelta
 from collections import Counter
 from argparse import ArgumentParser
 from configparser import ConfigParser, Error as ConfigParserError, NoSectionError, NoOptionError
-try:
-    from elasticsearch1 import VERSION as es_VERSION, Elasticsearch
-except ImportError:
-    from elasticsearch import VERSION as es_VERSION, Elasticsearch
-
-from pbench import tstos, es_index, es_put_template, PbenchConfig, BadConfig, get_pbench_logger
-
-
-_VERSION_ = "0.7.0.0"
-_NAME_    = "index-pbench"
-_DEBUG    = 0
+from pbench import tstos, get_es, es_index, es_put_template, PbenchConfig, BadConfig, get_pbench_logger
+
+
+_NAME_ = "index-pbench"
+_PARENT_ = "pbench-index"
+
+# This is the version of this python code. We use the version number to mean
```

Comment:
I remove the blank line in the comment so that it is all one paragraph, and reworked the text to kill the run-on sentence.  Since all the other comments go before the item that talk about, it seems this makes it more uniform keeping it before `_VERSION_` is defined.